### PR TITLE
some additions

### PR DESCRIPTION
**Pull request by olhabn**
performance screen clean and cleanAll buttons fixed

### CODE REVIEW

File: src/foam/u2/view/UnstyledTableView.js [hunk 1]
```diff
@@ -253,6 +253,10 @@ foam.CLASS({
           return foam.nanos.column.ColumnConfigToPropertyConverter.create();
         return this.__context__.columnConfigToPropertyConverter;
       }
+    },
+    {
+      name: 'onDAOUpdate',
```
Comment (kgrgreer) [inline src/foam/u2/view/UnstyledTableView.js]: Perhaps a better choice of name. Normally I would expect something named onXYZ to be a callback/listener.

File: src/foam/u2/view/UnstyledTableView.js [hunk 2]
```diff
@@ -411,7 +418,7 @@ foam.CLASS({
 
           //with this code error created  slot.get cause promise return
           //FIX ME
-          return this.slot(function(data, data$delegate, order, updateValues) {
+          return this.slot(function(data, data$delegate, onDAOUpdate, order, updateValues) {
```
Comment (kgrgreer) [inline src/foam/u2/view/UnstyledTableView.js]: Is there a better way to do this? Using data$proxy instead, maybe?

File: src/foam/u2/view/UnstyledTableView.js (hunk 1 above)
Comment (olhabn) [reply src/foam/u2/view/UnstyledTableView.js]: renamed

File: src/foam/u2/view/UnstyledTableView.js (hunk 2 above)
Comment (olhabn) [reply src/foam/u2/view/UnstyledTableView.js]: hmmm, probably, I don't have any suggestions as for now though. for some reason data$proxy doesn't work

Comment (olhabn) [reply src/foam/u2/view/UnstyledTableView.js]: we're using this approach in TableView

File: src/foam/u2/view/ReciprocalSearch.js [hunk 3]
```diff
@@ -150,6 +150,10 @@ foam.CLASS({
       var self = this;
 
       this.dao.on.sub(this.updateTotalCount);
+      this.dao.on.sub(function() {
```
Comment (olhabn) [inline src/foam/u2/view/ReciprocalSearch.js:153]: selectedCount wasn't updated on clearAll so I thought maybe this change will make some sense

**Pull request merged**